lib/hawkular/operations/operations_api.rb

```diff
@@ -53,14 +53,29 @@ def to_msg_hash
     # @example
     #   Hawkular::Operations::OperationsClient.new(credentials: {username: 'jdoe', password: 'password'})
     def initialize(args)
-      args[:host] ||= 'localhost:8080'
+      ep = args[:entrypoint]
```

Comment:
ok, thanks
there's no javadoc in a ruby client :wink: 
the lack of proper client documentation is really an obstacle...